include/xtensa-tdep.h

```diff
@@ -69,7 +71,7 @@ typedef enum
   xtRegisterGroupCP4	    = 0x10000000,    /* CP4.  */
   xtRegisterGroupCP5	    = 0x20000000,    /* CP5.  */
   xtRegisterGroupCP6	    = 0x40000000,    /* CP6.  */
-  xtRegisterGroupCP7	    = 0x80000000,    /* CP7.  */
+  xtRegisterGroupCP7	    = (int)0x80000000    /* CP7.  */
```

Comment:
Why is this needed? This is different from what's in the binutils gdb/xtensa-tdep.h and this file is supposed to be copied to gdb, it's unused otherwise.

Reply:
Oops, it appeared accidentally. This file is synced now with binutils-gdb master branch.